components/layout/construct.rs

```diff
@@ -250,13 +250,17 @@ impl InlineFragmentsAccumulator {
 
     fn from_inline_node<N>(node: &N, style_context: &SharedStyleContext) -> InlineFragmentsAccumulator
             where N: ThreadSafeLayoutNode {
+        let style = node.style(style_context);
+        let text_style = style_context.stylist.style_for_anonymous_box(
```

Comment:
If we're going to style text with this method, I'd rather rename it to something more generic that doesn't lie, like `pseudo_style` or something.